migrate/data/migrations/1701872471_initialise_schema.up.sql

```diff
@@ -1,6 +1,6 @@
 CREATE EXTENSION IF NOT EXISTS postgis;
 
-CREATE TABLE time_series (
+CREATE TABLE IF NOT EXISTS time_series (
```

Comment:
The `IF NOT EXISTS` should not be needed, if the migration framework is working correctly?

Reply:
True, added it as I thought it would not hurt, but removed it again.